README.md

```diff
@@ -1,5 +1,8 @@
 # Elitzur Data Validation
 
+https://img.shields.io/badge/lifecycle-production-1ed760.svg
```

Comment:
This isn't embedded properly I don't think.  This is what I see in preview
<img width="602" alt="Screen Shot 2020-05-04 at 12 42 07 PM" src="https://user-images.githubusercontent.com/13409909/80990679-bd96a000-8e04-11ea-878a-5d6edf5840c0.png">